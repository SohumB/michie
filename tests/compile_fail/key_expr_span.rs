use caching::caching;

#[caching(key_expr = b)]
fn f(a: bool) -> bool {
    a
}

fn main() {}
