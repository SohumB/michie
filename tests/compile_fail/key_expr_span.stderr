error[E0425]: cannot find value `b` in this scope
 --> tests/compile_fail/key_expr_span.rs:3:22
  |
3 | #[caching(key_expr = b)]
  |                      ^ help: a local variable with a similar name exists: `a`
